.github/workflows/cintegration-ecs-caller.yml

```diff
@@ -0,0 +1,36 @@
+name: CI ECS caller workflow
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches:
+    - main
+
+  pull_request:
+   branches: [main]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  call-reusable-workflow:
+    name: Graasp Explorer
```

Comment:
Actually is the CI pipeline for the Graasp Explorer, which implies that it will be deployed in dev. Once it ends something like this will appear: 
<img width="344" alt="Captura de pantalla 2022-04-14 a las 16 44 37" src="https://user-images.githubusercontent.com/36533965/163414902-8a0fe888-9a29-4990-90aa-1a141c2b7cd1.png">
 last step outputs "deploy to dev"